static/src/stylesheets/module/site-messages/_subscription-medium-banner.scss

```diff
@@ -0,0 +1,304 @@
+*[class^='site-message--subscription-banner'] {
+    box-sizing: border-box;
+}
+
+.site-message--subscription-banner {
+    position: relative;
+    display: flex;
+    flex-wrap: wrap;
+    overflow: hidden;
+    background: $highlight-main;
+    color: $brightness-7;
+    width: 100%;
+    padding: 10px;
+
+    @include mq($from: tablet) {
+        padding: 20px;
+    }
+}
+
+.site-message--subscription-banner__inner {
+    box-sizing: border-box;
+    display: flex;
+    flex-wrap: wrap;
+    position: relative;
+    margin: 0 auto;
+    max-width: 980px;
+
+    @include mq($from: leftCol) {
+        max-width: 1140px;
+    }
+
+    @include mq($from: wide) {
+        max-width: 1300px;
+    }
+}
+
+.site-message--subscription-banner__title {
+    font-size: 24px;
+    line-height: 24px;
+    font-family: $f-serif-headline;
+    font-weight: bold;
+    width: 100%;
+
+    @media (min-width: 600px) {
+        font-size: 28px;
+        line-height: 28px;
+    }
+
+    @include mq($from: tablet) {
+        margin-bottom: 10px;
+        width: 60%;
+        font-size: 4.7vw;
+        line-height: 4.9vw;
+    }
+
+    @include mq($from: desktop) {
+        width: 45%;
+        margin-right: 55%;
+        font-size: 34px;
+        line-height: 36px;
+    }
+}
+
+// text copy
+.site-message--subscription-banner__description {
+    display: flex;
+    align-items: flex-end;
+    align-content: stretch;
+    width: 50%;
+
+    @include mq($from: tablet) {
+        width: 60%;
+    }
+
+    @include mq($from: desktop) {
+        width: 45%;
+    }
+
+    p {
+        font-size: 17px;
+        line-height: 18px;
+        margin: 0;
+        font-family: $f-serif-text;
+
+        @include mq($from: mobileLandscape) {
+            font-size: 18px;
+            line-height: 20px;
+        }
+
+        @media (min-width: 550px) {
+            font-size: 20px;
+            line-height: 22px;
+        }
+
+        @include mq($from: phablet) {
+            font-size: 22px;
+            line-height: 24px;
+        }
+
+        @include mq($from: tablet) {
+            font-size: 2.5vw;
+            line-height: 3vw;
```

Comment:
Is there a reason these are in vw? These are quite awkward measurements, although the potential issues are limited here because there are media queries for the larger screen sizes. Just wondering about the rationale?

Reply:
I think it was just to keep it fluid through that med range of the view